httpx/_config.py

```diff
@@ -128,11 +127,10 @@ def load_ssl_context_verify(self) -> ssl.SSLContext:
 
         # Signal to server support for PHA in TLS 1.3. Raises an
         # AttributeError if only read-only access is implemented.
-        if sys.version_info >= (3, 8):  # pragma: no cover
-            try:
-                context.post_handshake_auth = True
-            except AttributeError:  # pragma: no cover
-                pass
```

Comment:
Why are you removing 3.8 conditionals?

Reply:
Python 3.7 no longer supported https://github.com/encode/httpx/pull/2813

Reply:
Yep, with 3.8+, the condition `sys.version_info >= (3, 8)` is always true.